fuzzers/libfuzzer_vartorcw/builder.Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG parent_image
+FROM $parent_image
+
+RUN git clone https://github.com/CodeIntelligenceTesting/llvm-project.git /llvm-project && \
```

Comment:
Does this have any other custom changes, can we just use the main llvm-project ?

Reply:
This is checkout is referencing [this commit](https://github.com/CodeIntelligenceTesting/llvm-project/commit/bbe1270aebdbd966d027d93c4a876499ac1573a9) in the CodeIntelligenceTesting/llvm-project repo. That commit contains my changes and its parent is the commit currently used by the `libfuzzer` core fuzzer.